db/schema.rb

```diff
@@ -22,27 +22,27 @@
     t.string   "job_title"
     t.datetime "created_at"
     t.datetime "updated_at"
-    t.integer  "veteran_id",   index: {name: "index_affiliations_on_veteran_id"}
+    t.integer  "veteran_id",   :index=>{:name=>"index_affiliations_on_veteran_id"}
   end
 
   create_table "awards", force: :cascade do |t|
     t.string   "title"
     t.string   "organization"
     t.datetime "created_at"
     t.datetime "updated_at"
-    t.integer  "veteran_id",   index: {name: "index_awards_on_veteran_id"}
+    t.integer  "veteran_id",   :index=>{:name=>"index_awards_on_veteran_id"}
```

Comment:
They were caused by the upgrade of the schema_plus_index Gem. We were locked on a forked version created by Mike Auclair, but upgrading rails caused that not to work and the bugfix in Mike's fork got incorporated into the official Gem.